src/translation/translation_de_DE.ts

```diff
@@ -30,17 +30,17 @@
     <message>
         <location filename="../util.cpp" line="597"/>
         <source>Qt cross-platform application framework</source>
-        <translation>Qt plattformübergreifender Anwendungsrahmen</translation>
+        <translation>Qt plattformübergreifendes Framework</translation>
     </message>
     <message>
         <location filename="../util.cpp" line="597"/>
         <source>(build)</source>
-        <translation type="unfinished"></translation>
+        <translation>(build)</translation>
```

Comment:
OK, so Google says "build from source" is "aus dem Quellcode erstellen", so I'd expect an abstract noun from the "erstellen" root verb, rather than the English?

Reply:
While not being German, "build" should be understood.